internal/command/user_human_profile_model.go

```diff
@@ -134,3 +122,25 @@ func (wm *HumanProfileWriteModel) NewChangedEvent(
 	}
 	return changeEvent, true, nil
 }
+
+func humanProfileWriteModelQuery(resourceOwner, userID string) *eventstore.SearchQueryBuilder {
```

Comment:
Please also use at least instanceID as the restriction is limited on instance

Reply:
will revert that.

Reply:
removed